engine/security/rootless.md

```diff
@@ -345,10 +345,11 @@ Add `net.ipv4.ping_group_range = 0   2147483647` to `/etc/sysctl.conf` (or
 
 ### Exposing privileged ports
 
-To expose privileged ports (< 1024), set `CAP_NET_BIND_SERVICE` on `rootlesskit` binary.
+To expose privileged ports (< 1024), set `CAP_NET_BIND_SERVICE` on `rootlesskit` binary and restart the daemon.
 
 ```console
-$ sudo setcap cap_net_bind_service=ep $HOME/bin/rootlesskit
+$ sudo setcap cap_net_bind_service=ep /usr/bin/rootlesskit
```

Comment:
The path depends on installation method, so $(which rootlesskit) might be more robust

Reply:
Thank you @AkihiroSuda. @koelle25 Could you please update as per Akihiro's suggestion above?